features/visual-search.md

```diff
@@ -0,0 +1,69 @@
+
+By searching visually rather than by name or metadata, you can:
+
+* Easily locate images without needing to remember the tags or names of the files.
+* Search for similar images in your media library.
+* Identify duplicates in your media library with ease.
+* Discover images in your media library through visual characteristics.
+* Group visually similar images together for better organization.
+* Enhance the discoverability of your assets.
+
+You can perform visual search from the dashboard either by text or by image:
+
+*   **Searching by Text**: You can type in a word or a phrase to find images that visually match it.\
+    For example, you can type in `pink birds`; all the images related to `pink birds` will be retrieved from your media library.\
+
+
+    <figure><img src="../.gitbook/assets/text_search.png" alt=""><figcaption></figcaption></figure>
+
+    <figure><img src="../.gitbook/assets/visual-search/text_search_results.png" alt=""><figcaption></figcaption></figure>
+*   **Searching by Image**: You can find matches based on the image you upload.\
+
+    <figure><img src="../.gitbook/assets/visual-search/search_by_image.png" alt=""><figcaption><p>Upload image or specify a URL</p></figcaption></figure>
+
+    For example, uploading the image from [this URL](https://ik.imagekit.io/pwliscd3n/photo-1491972690050-ba117db4dc09\_9\_0DBlXmS.jpeg) will retrieve images related to a woman wearing a red shirt with sunglasses.\
+
+
+    <figure><img src="../.gitbook/assets/visual-search/image_search_results.png" alt=""><figcaption></figcaption></figure>
+*   **Searching by Image URL**: You can find matches based on the image URL you provide.\
+
+
+    <figure><img src="../.gitbook/assets/visual-search/search_by_image_url.png" alt=""><figcaption><p>Search by Image URL</p></figcaption></figure>
+
+
+
+    For example, entering [this URL](https://ik.imagekit.io/pwliscd3n/allison-huang-\_u8KhAZRGHs-unsplash\_rn6D9ksWC.jpg) in the input box will retrieve images related to a scenic view of a beach with a house.\
+
+
+    <figure><img src="../.gitbook/assets/visual-search/image_url_search_results.png" alt=""><figcaption></figcaption></figure>
+
+## Finding Similar Images to assets uploaded in the Media Library
+
+Visual Search lets you easily and quickly identify duplicate assets or find similar images in your media library.\
+\
+You can do this directly from the File Details page in the Media Library.
+
+<figure><img src="../.gitbook/assets/visual-search/file_details_page.png" alt=""><figcaption></figcaption></figure>
+
+Use the "Find Similar" button to execute a visual search using the image URL.
+
+<figure><img src="../.gitbook/assets/visual-search/similar_image_search.png" alt=""><figcaption></figcaption></figure>
+
+The search results will include all images that match the context of the image you are trying to find similar ones to. As seen in the example above, the first couple of results will likely be duplicates.
```

Comment:
"the first couple of results will likely be duplicates.", this is not necessarily true.